scripts.py

```diff
@@ -1323,16 +1331,33 @@ def do_run(self, cmd_args=None):
         ebook_directory = parsed.ebook_directory
         rights_uri = parsed.rights_uri
         dry_run = parsed.dry_run
+        default_medium_type = parsed.default_medium_type
+
         return self.run_with_arguments(
-            collection_name, collection_type, data_source_name,
-            metadata_file, metadata_format, cover_directory,
-            ebook_directory, rights_uri, dry_run
+            collection_name,
+            collection_type,
+            data_source_name,
+            metadata_file,
+            metadata_format,
+            cover_directory,
+            ebook_directory,
+            rights_uri,
+            dry_run,
+            default_medium_type
```

Comment:
It's up to you, but at this point when I'm adding yet another argument to a method that takes a lot of arguments, I usually switch over to passing in keyword arguments. This reduces the risk that I'll mix up the order of two arguments and create a difficult-to-find bug.

Reply:
It's a good advice, changed the code accordingly